core/nilai.go

```diff
@@ -0,0 +1,28 @@
+package core
+
+import (
+	"context"
+
+	"github.com/tokopedia/code-review-workshop/errors"
+	"github.com/tokopedia/code-review-workshop/tracer"
+)
+
+type NilaiSiswa struct {
+	NIM     int64  `json:"nim"      db:"nim"`
+	MapelID int64  `json:"mapel_id" db:"mapel_id"`
+	Nilai   int64  `json:"nili"     db:"nili"`
```

Comment:
```suggestion
	Nilai   float32  `json:"nilai"     db:"nilai"`
```
nilai biasanya ada angka di belakang koma